tests_integration/runPrettier.js

```diff
@@ -101,7 +104,8 @@ function runPrettier(dir, args, options) {
     jest.restoreAllMocks();
   }
 
-  const result = { status, stdout, stderr, write };
+  const formatCalls = formatSpy.mock.calls.length;
+  const result = { status, stdout, stderr, write, formatCalls };
```

Comment:
What about returning below:

```js
return Object.assign({ test: testResult, formatCalls }, result);
```

and then write one or more tests that assert specifically the number of format calls:

```js
const result = runPrettier(args);
expect(result.formatCalls).toEqual(0);
```

WDYT?

Reply:
Sounds good to me! I'll check it out tomorrow :)